runtime/doc/health.txt

```diff
@@ -21,6 +21,19 @@ To run all healthchecks, use: >vim
 <
 Plugin authors are encouraged to write new healthchecks. |health-dev|
 
+                                                                    *g:health*
+g:health  This global variable controls the behavior and appearance of the
+          `health` floating window. It should be a dictionary containing the
+          following optional keys:
+          - `style`: string? Determines the display style of the `health` window.
+                            Set to `'float'` to enable a floating window. Other
+                            styles are not currently supported.
+          - `max_height`: integer? Max height of floating window
+          - `max_width`: integer? Max width of floating window
```

Comment:
Want to avoid surface area where possible. max_height/width should be dropped for now. Just choose something reasonable.

Reply:
Just thought some people might have some fixed settings for height and width.  cleaned up